industry_benchmarks/utils/extras/extract_results.py

```diff
@@ -96,6 +96,22 @@ def extract(results_0, results_1, results_2, output):
     files_0 = glob.glob(f"{results_0}/*.json")
     files_1 = glob.glob(f"{results_1}/*.json")
     files_2 = glob.glob(f"{results_2}/*.json")
+    # Check to make sure all the json files are result json files
+    # TODO We can combine this with the error checking loop. I am
+    # not sure the order to do this in, since I thought the checks
+    # before "checking files for errors" section would fail fast but
+    # we are seeing users have 1 extra json file in each folder,
+    # which then is tripping things up later
+    for file in tqdm(files_0 + files_1 + files_2):
+        with open(file, 'r') as fd:
+            results = json.load(fd)
+        # First we check if someone passed in an input json
+        if "name" in results.keys():
+            raise ValueError(f"{file} is an input json, move this file outside of the results directory")
```

Comment:
Rather than failing on individual files, it would be best to either skip (i.e. pop from the relevant list) or make a full list of files to be moved.

Personallly I'd prefer just do a pop call unless there's a good reason not to.